### PR TITLE
Exclude AinferNullnessAjavaTest from junit tests.

### PR DESCRIPTION
**Pull request by smillst**

### CODE REVIEW

**Comment by mernst** [comment]
What is the reason for this exclusion?  @kelloggm might be a better reviewer than I am.

File: checker/build.gradle [hunk 1]
```diff
@@ -374,6 +374,7 @@ test {
         excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferTestCheckerStubsTest'
         excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferTestCheckerAjavaTest'
         excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferNullnessJaifsTest'
+        excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferNullnessAjavaTest'
```
Comment (kelloggm) [inline checker/build.gradle:377]: This should also be added to the list of `ainferTest` tasks around line 840, so that it does get run.

**Review by kelloggm — CHANGES_REQUESTED** [review]
Thanks for catching my mistake: I should have added this exclusion when I added the test.

File: checker/build.gradle (hunk 1 above)
Comment (smillst) [reply checker/build.gradle:377]: Is that in addition to this change?

**Review by kelloggm — APPROVED** [review]
LGTM

**Pull request merged**